scripts/build-styles.js

```diff
@@ -38,7 +38,7 @@ async function sassRender(sourceFile, templateFile, outputFile) {
 
 (async () => {
   const template = path.resolve(process.argv[1], '../styles.tmpl');
-  const paths = await globby(['src/components/**/*.scss', 'src/styles/themes/**/*.scss']);
+  const paths = await globby(['src/components/**/*.material.scss']);
```

Comment:
Is this applicable to material only?

Reply:
Yes, the other component theme files are only used in the global theme files.